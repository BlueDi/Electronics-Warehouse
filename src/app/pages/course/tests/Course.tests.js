import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';
import Course from '@pages/course';

describe('<Course />', () => {
  test('Renders properly', () => {
    const component = renderer.create(
      <Router>
        <Course />
      </Router>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
