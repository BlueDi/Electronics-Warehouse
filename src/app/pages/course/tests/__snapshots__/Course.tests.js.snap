// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Course /> Renders properly 1`] = `null`;
